src/testing/run.ts

```diff
@@ -244,11 +244,15 @@ export async function runTestSuite<
       throw new Error(`[${args.id}] No test cases provided.`);
     }
     args.evaluators.forEach((evaluator) => {
-      if (!(evaluator instanceof BaseTestEvaluator)) {
```

Comment:
we could have left this if BaseEvaluator extends BaseTestEvaluator but it's just a separate class